docs/releases/unreleased.md

```diff
@@ -8,6 +8,11 @@
 
 - The `predict_many` method scikit-learn models wrapped with `compat.convert_sklearn_to_river` raised an exception if the model had been fitted on any data yet. Instead, default predictions will be produced, which is consistent with the rest of River.
 
+## neighbors
+
+- Add SWINN to power-up approximate nearest neighbor search. SWINN uses graphs to speed up nearest neighbor search in large sliding windows of data.
+- Add classes `ANNClassifier` and `ANNRegressor` to perform approximate nearest neighbor search in classification and regression tasks.
```

Comment:
```suggestion
- Add `neighbors.SWINN` to power-up approximate nearest neighbor search. SWINN uses graphs to speed up nearest neighbor search in large sliding windows of data.
- Add classes `neighbors.ANNClassifier` and `neighbors.ANNRegressor` to perform approximate nearest neighbor search in classification and regression tasks.
```

Tip: adding the full path will automatically add a link when the doc is rendered :)